plugins/thunderbolt/fu-plugin-thunderbolt.c

```diff
@@ -75,4 +101,6 @@ fu_plugin_init_vfuncs(FuPluginVfuncs *vfuncs)
 	vfuncs->startup = fu_plugin_thunderbolt_startup;
 	vfuncs->device_registered = fu_plugin_thunderbolt_device_registered;
 	vfuncs->device_created = fu_plugin_thunderbolt_device_created;
+	vfuncs->composite_prepare = fu_plugin_composite_prepare;
+	vfuncs->composite_cleanup = fu_plugin_composite_cleanup;
```

Comment:
you should include the word `thunderbolt` in these like other vfuncs to avoid potential symbol collisions when loaded into the daemon.